pypeit/par/pypeitpar.py

```diff
@@ -497,7 +497,7 @@ def __init__(self, method=None, pixelflat_file=None, spec_samp_fine=None,
                  spec_samp_coarse=None, spat_samp=None, tweak_slits=None, tweak_slits_thresh=None,
                  tweak_slits_maxfrac=None, rej_sticky=None, slit_trim=None, slit_illum_pad=None,
                  illum_iter=None, illum_rej=None, twod_fit_npoly=None, saturated_slits=None,
-                 slit_illum_relative=None):
+                 slit_illum_relative=None, ref_idx=None):
```

Comment:
How about renaming ref_idx to  slit_illum_ref_idx to make it more clear that this parameter is associated with that option.

Reply:
Done!